R/install.R

```diff
@@ -244,7 +244,7 @@
 }
 
 .install_updated_version <-
-    function(valid, update, repos, ...)
+    function(valid, update, old_pkgs, instPkgs, repos, force, ...)
```

Comment:
I don't think adding `force` here and in line 257 is necessary, it'll just be passed through the `...`. Line 432 should then have `force = force`.